clang/lib/Driver/ToolChains/Flang.cpp

```diff
@@ -866,6 +866,17 @@ void Flang::ConstructJob(Compilation &C, const JobAction &JA,
     }
   }
 
+  // -fno-strict-overflow implies -fwrapv if it isn't disabled, but
+  // -fstrict-overflow won't turn off an explicitly enabled -fwrapv.
+  if (Arg *A = Args.getLastArg(options::OPT_fwrapv, options::OPT_fno_wrapv)) {
+    if (A->getOption().matches(options::OPT_fwrapv))
+      CmdArgs.push_back("-fwrapv");
+  } else if (Arg *A = Args.getLastArg(options::OPT_fstrict_overflow,
+                                      options::OPT_fno_strict_overflow)) {
+    if (A->getOption().matches(options::OPT_fno_strict_overflow))
+      CmdArgs.push_back("-fwrapv");
+  }
+
```

Comment:
If this code is exactly the same as it is in clang, it would be better if it is shared. In `clang/lib/Driver/ToolChains/CommonArgs.cpp`, see `handleColorDiagnosticsArgs` for a case of common argument handling functionality between clang and flang. This was done in #109210

Reply:
Thank you for showing me a better way.
I created a function `renderIntegerOverflowOptions` to handle common options between clang and flang. However, I'm not sure the name of the function is appropriate because there are other related options handled only by clang (e.g. `-ftrapv`).

Reply:
Perhaps the function could be named `renderCommonIntegerOverflowOptions` which suggests that it does not handle all overflow options?

Reply:
That sounds good. I fixed it. Thank you.